onions.md

```diff
@@ -2,26 +2,26 @@
 
-ONIONS will also develop recommendations for operationalizing YANG-based APIs (YANG2API). This involves investigating and defining the components required to programmatically transform YANG modules into open API specifications for easier consumption by operators and systems, as well as assessing the applicability of abstractions to accommodate specific operational use cases and deployments, ensuring relevance to real-world environments.
+ONIONS will also develop recommendations for operationalizing YANG-based APIs (YANG2API). This involves investigating and defining the requirements to programmatically transform YANG modules into open API specifications for easier consumption by operators and systems. Additionally, the WG will assess the applicability of these abstractions to accommodate specific operational use cases and deployments, ensuring relevance to real-world environments.
 
-The ONIONS WG will evaluate implementation approaches and perform gap analyses of intended abstractions by assessing interactions with control and data plane features necessary to implement services within networks and identifying and addressing any gaps in the implementation of these abstractions.
+The ONIONS WG will evaluate implementation approaches and perform gap analyses of intended abstractions. This involves assessing interactions with control and data plane features necessary to implement services within networks. The WG will identify and work towards addressing any identified gaps in the implementation of these abstractions, coordinating with relevant working groups as necessary.
 
 Furthermore, ONIONS will coordinate with the Routing Area and other relevant WGs to evaluate operational needs of abstraction-driven solutions (e.g., slicing). This will involve facilitating two-way coordination to ensure efficient alignment of efforts.
 
-The ONIONS WG will produce recommendations and tools to operationalize YANG APIs (YANG2API); updates to L2NM and L3NM models based on feedback and deployment experiences (especially considering the need for operational state data); assessments and recommendations for the applicability of AC data models in hybrid network/cloud environments; approaches for mapping between abstraction layers, ensuring seamless interoperability; and guidelines for integrating IETF YANG models with external frameworks (e.g., TMF640). The ONIONS WG will also engage in hackathons and community-driven validation efforts to test and refine its outputs.
+The ONIONS WG will produce recommendations for tools using YANG-based APIs (YANG2API), updates to L2NM and L3NM models, assessments and recommendations for attachment circuits (AC) data models, approaches for mapping between abstraction layers, and guidelines for integrating IETF YANG models with external frameworks (e.g., TMF640). The ONIONS WG will also engage in hackathons and community-driven validation efforts to test and refine its outputs and produce proof-of-concept YANG2API tools.
```

Comment:
> The ONIONS WG will also engage in hackathons and community-driven validation efforts to test and refine its outputs and produce proof-of-concept YANG2API tools.

The first part of the sentence emphasizes engagement, but the latter section suggests that the WG will also produce (and take ownership of the PoC). Lets rephrase the latter?